src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -719,7 +719,9 @@ public function delete($id, $checkSecurity = false)
                     // this will trigger massive-search deindex
                     $this->em->remove($fileVersionMeta);
                 }
+                $this->em->detach($fileVersion);
```

Comment:
Isn't that something that should always happen, no matter how the medium is deleted? Because I think that is something that should go into the Doctrine configuration (the `.orm.xml` files).